app-android/src/main/java/io/github/droidkaigi/confsched2023/KaigiApp.kt

```diff
@@ -284,6 +292,14 @@ private class ExternalNavController(
         context.startActivity(Intent(context, OssLicensesMenuActivity::class.java))
     }
 
+    fun onShareClick(timeTableItem: TimetableItem) {
+        shareNavigator.share(
+            "[${timeTableItem.room.name.currentLangTitle}] ${timeTableItem.startsTimeString} - ${timeTableItem.endsTimeString}\n" +
```

Comment:
How about adding date info too? 👀
Perhaps, we are referring the formatted date property on the detail screen, so we may reuse it.

Reply:
@tkhs0604 
It seems that we can add a method to get only month and date by adding it to TimetableItem.kt.
However, I felt it was a bit redundant to add a method that is only used for the share function, so I decided to revert to the original method. :bow: